### PR TITLE
Guidance images

### PR DESCRIPTION
**Pull request by mviegasb**
- updated and reordered content to be consistent with how GOV.UK patterns are documented:
(When to use it & when not, how to use it, examples, research, accessibility, updated 'contact box
- updated examples & included which services used in 

- removed previous example used in passports as they have been removed from the service following further testing which showed they did not help the user journey.

### CODE REVIEW

File: app/views/patterns/guidance-image.html [hunk 1]
```diff
@@ -16,43 +16,116 @@
...
+      <p>The examples below have been used in public-facing services. Do you have examples of using an image for guidance in an internal service? Get in touch at <a href="mailto:designops@digital.homeoffice.gov.uk">designops@digital.homeoffice.gov.uk.</a></p>
+
+      <h3 id="biometric-residence-permit" class="govuk-heading-m">Biometric residence permit</h3>
+        <div class="example example-images confirmation" style="margin-bottom: 2em;">
+          <img src="{{ asset_path }}/images/image-guidance-brp.svg" class="confirm-desktop" alt="An example of an illustrative image of a biometric residence permit card">
+          <img src="{{ asset_path }}/images/image-guidance-brp.svg" class="confirmation confirm-mobile" alt="An example of an illustrative image of a biometric residence permit card">
+        </div>
+        <h4 class="govuk-heading-s">Services using this image</h4>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>Right to work</li>
```
Comment (karypun) [inline app/views/patterns/guidance-image.html:53]: could we keep this in alphabetical order?

File: app/views/patterns/guidance-image.html [hunk 2]
```diff
@@ -16,43 +16,116 @@
...
+      <br>
 
+      <h3 id="machine-readable-zone-card" class="govuk-heading-m">ID card with a Machine Readable Zone</h3>
+      <div class="example example-images confirmation" style="margin-bottom: 2em;">
+        <img src="{{ asset_path }}/images/image-guidance-mrz-card.svg" class="confirm-desktop" alt="An example of a guidance image used on a service, with text on the top and an illustrative image below it">
+        <img src="{{ asset_path }}/images/image-guidance-mrz-card.svg" class="confirmation confirm-mobile" alt="An example of a guidance image used on a service, with text on the top and an illustrative image below it">
+      </div>
+      <h4 id="related" class="govuk-heading-s">Services using this image</h4>
+      <ul class="govuk-list govuk-list--bullet">
+      <li><a href="https://www.gov.uk/guidance/using-the-eu-exit-id-document-check-app">EU Exit: ID Document Check app</a></li>
```
Comment (karypun) [inline app/views/patterns/guidance-image.html:100]: Could we put this example next to the BRP one to tie in Immigration services?

File: app/views/patterns/guidance-image.html [hunk 3]
```diff
@@ -16,43 +16,116 @@
...
+      <h3 id="biometric-residence-permit" class="govuk-heading-m">Biometric residence permit</h3>
+        <div class="example example-images confirmation" style="margin-bottom: 2em;">
+          <img src="{{ asset_path }}/images/image-guidance-brp.svg" class="confirm-desktop" alt="An example of an illustrative image of a biometric residence permit card">
+          <img src="{{ asset_path }}/images/image-guidance-brp.svg" class="confirmation confirm-mobile" alt="An example of an illustrative image of a biometric residence permit card">
+        </div>
+        <h4 class="govuk-heading-s">Services using this image</h4>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>Right to work</li>
+        <li>Refugee integration loan</li>
+        <li>Right to rent</li>
```
Comment (karypun) [inline app/views/patterns/guidance-image.html:55]: Right to work: https://www.gov.uk/prove-right-to-work
The other links we will need the teams to verify the links.

File: app/views/patterns/guidance-image.html (hunk 1 above)
Comment (mviegasb) [reply app/views/patterns/guidance-image.html]: Sure, have updated this!

**Comment by mviegasb** [comment]
I've made the requested changes and updated the order of the examples. @MarionDownie would be great to get your eyes on this too, thank you!

File: app/views/patterns/guidance-image.html [hunk 4]
```diff
@@ -24,7 +24,7 @@ <h2 id="when-to-use-this-pattern" class="govuk-heading-m">When to use this patte
         <li>complex guidance when completing a task</li>
         <li>how to perform an action</li>
         <li>what information you’re asking from them </li>
-        <li>where to find specific contextual information, such as a reference number on a document or the specific side of an ID document</li>
+        <li>where to find specific contextual information, such as a reference number on a document or the specific side of an identity document</li>
```
[the remark this answers is not in the document]
Comment (mviegasb) [reply app/views/patterns/guidance-image.html]: Have done, thanks!

File: app/views/patterns/guidance-image.html [hunk 5]
```diff
@@ -35,13 +35,13 @@ <h2 id="when-not-to-use-this-pattern" class="govuk-heading-m"=>When not to use t
...
       <h2 id="how-to-use-this-pattern" class="govuk-heading-m">How to use this pattern</h2>
-      <p class="govuk-body">If the image represents something physical, such as a document, you should use the aspect ratio of that object. Follow our <a href="/styles/images">guidance on the use of images.</a></p>
+      <p class="govuk-body">If the image represents something physical, such as a document, you should use the aspect ratio of that object. Follow our guidance on the use of <a href="/styles/images">images</a>.</p>
 
       <br>
 
       <h2 id="examples" class="govuk-heading-l">Examples</h2>
 
-      <p>The examples below have been used in public-facing services. Do you have examples of using an image for guidance in an internal service? Get in touch at <a href="mailto:designops@digital.homeoffice.gov.uk">designops@digital.homeoffice.gov.uk.</a></p>
+      <p>The examples below have been used in public-facing services. Do you have examples of using an image for guidance in an internal service? Get in touch at <a href="mailto:designops@digital.homeoffice.gov.uk">designops@digital.homeoffice.gov.uk</a>.</p>
```
[the remark this answers is not in the document]
Comment (mviegasb) [reply app/views/patterns/guidance-image.html]: Have done!

File: app/views/patterns/guidance-image.html [hunk 6]
```diff
@@ -16,30 +16,32 @@
     <div class="app-prose-scope">
       <span class="govuk-caption-xl">Patterns</span>
       <h1 class="heading-section">Guidance images</h1>
-      <p class="govuk-body-l">Help users understand a complex message, show a user how to perform an action or guide a user to complete a task.</p>
+      <p class="govuk-body-l">Using guidance images can help users to understand a complex message, show a user how to perform an action or guide a user to complete a task.</p>
 
-      <h2 id="when-to-use-this-pattern" class="govuk-heading-m">When to use guidance images</h2>
-      <p class="govuk-body">Only use images when research identifies a user need and to enhance the information being conveyed. Used correctly, images can help users to understand:</p>
+      <h2 id="when-to-use-this-pattern" class="govuk-heading-m">When to use this pattern</h2>
+      <p class="govuk-body">Only use guidance images when research indicates a real user need and to enhance the information being conveyed. Used correctly, guidance images can help users understand:</p>
```
[the remark this answers is not in the document]
Comment (mviegasb) [reply app/views/patterns/guidance-image.html]: Reads much better, thank you.

File: app/views/patterns/guidance-image.html [hunk 7]
```diff
@@ -16,43 +16,96 @@
...
       <ul class="govuk-list govuk-list--bullet">
-        <li><a href="https://home-office-digital-patterns.herokuapp.com/patterns/interruption-card">Interruption card</a></li>
-        <li><a href="https://design-system.service.gov.uk/styles/images/">GOV.UK Design System images guidance</a></li>
+        <li><a href="/styles/images/">Home Office design system images guidance</a></li>
       </ul>
 
       <div class="contact-us">
         <h2 class="govuk-heading-m">Get in touch</h2>
-        <p>If you’ve got a question or suggestion share it on Slack <a href="slack://channel?id=CTXV6L1K4&team=T03TJ3P61">#ho-design-system channel</a> or on <a href="https://github.com/UKHomeOffice/home-office-digital-patterns/issues/92">GitHub</a>.</p>
+        <p>If you’ve got a question or suggestion share it on the Slack <a href="slack://channel?id=CTXV6L1K4&team=T03TJ3P61">#ho-design-system channel</a>, on <a href="https://github.com/UKHomeOffice/design-system" target=blank>GitHub</a> or email the design system working group on <a href="mailto:designops@digital.homeoffice.gov.uk">designops@digital.homeoffice.gov.uk</a>.</p>
```
[the remark this answers is not in the document]
Comment (mviegasb) [reply app/views/patterns/guidance-image.html]: That's fine - I have to go through all the pages and update that section anyway so can change that here too. Thanks!

**Comment by mviegasb** [comment]
> Hi Mariana, I've reviewed all the commits and made some minor extra suggestions.
> 
> Also one thing I couldn't seem to comment on - the image for the ID card looks slightly less sharp than the others. Not sure if there's anything we can do about that.

Yes - I'm waiting for the original source file, at the moment we only have a small image so it's a little fuzzy. The original file for the passport photo code isn't any better unfortunately.

**Comment by mviegasb** [comment]
Ah, I see what you mean! Have added channel back in, outside of the hyperlink. Thanks! :) @MarionDownie

**Pull request merged**